micrometer-core/src/main/java/io/micrometer/core/instrument/Observer.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micrometer.core.instrument;
+
+/**
+ * @author Dmitry Poluyanov
+ * @since 23.07.17
+ */
+public interface Observer {
```

Comment:
No, thats approach similarly to histogram, but in histogram we always need choose between size of buckets, that would be collected and accuracy of every value (mean size of bucket and count of buckets). 

I want to create timer implementation that will collect every record pair(timestamp/value), and send it to storage with given measure step or by reaching some batch size, and later use this metrics with one of [quantile*/median*](https://clickhouse.yandex/docs/en/agg_functions/index.html#quantile-level-x) CH function.
Raw measurements more useful for different aggregations, but i need more performance tests for graphic calculation queries and insertion queries. Now i have tests only with ~500 records per second on small number of timers (around 400000 sent measurements per minute) from small amount of application. 
Different approaches can be used in different places and it will be useful to be able to choose between them.